src/features/customerView/invoicingHistoryCard/InvoicingHistoryCard.tsx

```diff
@@ -37,37 +39,49 @@ const InvoicingHistoryCard = ({ invoices, onClick }: InvoicingHistoryProps) => {
     }
   };
 
-  const { t, i18n } = useTranslation();
+  const getRows = () =>
+    invoices.map((invoice, id) => (
+      <React.Fragment key={id}>
+        <button onClick={() => onClick(invoice)} className={styles.gridItem}>
+          <Text color="brand">
+            {isBerthInvoice(invoice)
+              ? t('common.terminology.berthRent')
+              : t('common.terminology.winterStoragePlaceRent')}
+          </Text>
+        </button>
+        <div className={styles.gridItem}>
+          <Text>
+            {`${formatDate(invoice.contractPeriod.startDate, i18n.language)} - ${formatDate(
+              invoice.contractPeriod.endDate,
+              i18n.language
+            )}`}
+          </Text>
+        </div>
+        <div className={styles.gridItem}>
+          <Text>{formatDate(invoice.dueDate, i18n.language)}</Text>
+        </div>
+        <div className={styles.gridItem}>
+          <Text>{formatPrice(invoice.totalPrice, i18n.language)}</Text>
+        </div>
+        <div className={styles.gridItem}>
+          <StatusLabel type={invoiceStatusToType(invoice.status)} label={t(getOrderStatusTKey(invoice.status))} />
+        </div>
```

Comment:
Do we need these wrappers? Cannot we assign the `className`s directly to the wrapped elements?
e.g.
```suggestion
        <StatusLabel className={styles.gridItem} type={invoiceStatusToType(invoice.status)} label={t(getOrderStatusTKey(invoice.status))} />
```

Reply:
The wrapper is needed, because otherwise the `StatusLabel` gets stretched to fill the cell
![image](https://user-images.githubusercontent.com/15479131/97844225-51cdbb00-1cf3-11eb-8b38-2fa586e20090.png)

This is the appearance with the wrapper
![image](https://user-images.githubusercontent.com/15479131/97844270-6611b800-1cf3-11eb-830b-c31df072559d.png)